naptime/src/main/scala/org/coursera/naptime/ari/engine/EngineHelpers.scala

```diff
@@ -109,6 +112,31 @@ object EngineHelpers extends StrictLogging {
     }
   }
 
+  // TODO(bryan): Dedupe this from NaptimeUnionField
```

Comment:
With the removal of `EngineImpl`, should this file (and possibly this package) be renamed? `utilities` or something similar?